ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/port-channel-interfaces.md

```diff
@@ -586,6 +586,7 @@ interface Port-Channel110
    description isis_interface_knobs
    no switchport
    isis enable ISIS_TEST
+   isis bfd
```

Comment:
Please add a BFD column to the ISIS interface tables for all interface types.

Reply:
Added